joomla/joomla-cms
# [4.4] Fix message for banner tracks delete

Pull Request for Issue #42700 .

### Summary of Changes
The issue contains two parts. 
1) A wrong message is displayed, it shows the number of existing tracks as deleted, not the deleted tracks
2) If the filter is not set properly and if there are tracks to non existing banners and categories, these tracks never are deleted. 

This PR is for 1) 
The number of deleted rows, not the total amount of rows is displayes


### Testing Instructions
You must have tracks in #__banner_tracks. 
Delete tracks and see which number is returned


### Actual result BEFORE applying this Pull Request
See the issue #42700
It says that the whole number of existing tracks is deleted


### Expected result AFTER applying this Pull Request
Only the number of affectedRows is displayed


### Link to documentations
No documentation, it'a bug
